cmd/bb_scheduler/build_queue_state_service.go

```diff
@@ -182,7 +226,7 @@ var (
 <!DOCTYPE html>
 <html>
   <head>
-    <title>All operations</title>
+    <title>{{operations_title .Selection}} operations {{present_if_value "for invocationID %s" .InvocationID}}</title>
```

Comment:
The `present_if_value` macro doesn't really carry its own weight. What's the advantage of this over `{{with .InvocationID}} for invocation {{.}}{{end}}`?